RELEASE/scripts/sl_ascend.ash

```diff
@@ -12737,8 +12737,18 @@ boolean L9_chasmBuild()
 		buffMaintain($effect[Carol of the Hells], 50, 1, 1);
 		buffMaintain($effect[Song of Sauce], 150, 1, 1);
 
-		// Since we are Buffing for Spells and Myst, set choice adventure just in case
-		set_property("choiceAdventure1345", 2);
+		if(get_property("sl_aggressiveBlechHouse"))
+		{
+			// Since we are Buffing for Spells and Myst, set choice adventure just in case
+			print("Aggressive Blech House Enabled. Set for option 2: Blast it down with a spell", "green");
+			set_property("choiceAdventure1345", 2);
+		}
+		else
+		{
+			print("If we encounter Blech House when we are not expecting it we will stop.", "blue");
+			print("Currently setup for Myst/Spell Damage, option 2: Blast it down with a spell", "red");
```

Comment:
This should probably be blue too. Although I'm not sure if we really need to print anything here at all?

Reply:
In case Blech House does happen they know what option is spec'd so they can manually choose for maximum effect.